### PR TITLE
fix: watch the ZK node once the master TaskManager has been changed.

### LINKED ISSUE #1066: Update TaskManager Java SDK to initialize with ZK and watch ZK node
Now the Java SDK of TaskManager access BRPC server directly and we should watch th ZK node once the master TaskManager has been changed.

### PR DESCRIPTION
**Pull request by yabg-shuai666**
close #1066

### CODE REVIEW

File: java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/client/TaskManagerClient.java [hunk 1]
```diff
@@ -37,6 +37,50 @@
...
+        String msg;
+        HostPort hostPort = new HostPort(TaskManagerConfig.HOST, TaskManagerConfig.PORT);
+        CuratorFramework zkClient = CuratorFrameworkFactory.builder() //
+                .connectString(zkCluster)   //连接地址
+                .sessionTimeoutMs(10000)    //超时时间
+                .retryPolicy(new ExponentialBackoffRetry(1000, 10))   //重试策略
+                .build();//
+        zkClient.start();
+
+        try {
```
Comment (aceforeverd) [inline java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/client/TaskManagerClient.java:63]: constructor should not contain heavy works, how about move to separate function `init()`

Comment (yabg-shuai666) [reply java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/client/TaskManagerClient.java:63]: I will put part of work into a separate init () function.

Comment (aceforeverd) [reply java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/client/TaskManagerClient.java:63]: I mean, do not do call `init` inside constructor.

**Comment by tobegit3hub** [comment]
Replaced by https://github.com/4paradigm/OpenMLDB/pull/1163 .

**Pull request closed without merging**